dist/tools/codespell/check.sh

```diff
@@ -10,7 +10,9 @@ CODESPELL_OPT=" -c"
 CODESPELL_OPT+=" -q 2"
 CODESPELL_OPT+=" --check-hidden"
 CODESPELL_OPT+=" --ignore-words ${TOOLS}/codespell/ignored_words.txt"
-CODESPELL_OPT+=" --skip=RIOT,dist,./wifi-subsys/build,.git,css,html,js,m4a.doxyfile"
+CODESPELL_OPT+=" --skip=RIOT,dist,build,bin,.git,css,html,js,m4a.doxyfile"
+CODESPELL_OPT+=" -D ${TOOLS}/codespell/dictionary.txt"
```

Comment:
Here is called the dictionary element, could be works using the default, but I added one dictionary to solve the unknown words and get the expected output locally